S/Spasm/bundled/patches/zap-gmp-test.patch

```diff
@@ -0,0 +1,30 @@
+diff --git givaro.orig/macros/gmp-check.m4 givaro/macros/gmp-check.m4
+--- givaro.orig/macros/gmp-check.m4
++++ givaro/macros/gmp-check.m4
+@@ -105,21 +105,11 @@ AC_DEFUN([GIV_CHECK_GMP], [
+ 		exit 1
+ 	])
+ 
+-	AC_MSG_CHECKING([whether gmp version is at least $min_gmp_release])
+-	AC_TRY_RUN(
```

Comment:
Someone should tell them that all these `AC_TRY_RUN` can be replaced with trivial compiler tests.